style.css

```diff
@@ -267,3 +267,170 @@ h5 {
     }
   }
 }
+
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  top: 72rem;
+  height: 44rem;
+  background-image: url(image/contact\ form\ shapes-mobile.png);
+  background-color: #396df2;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow-x: hidden !important;
+  border-top-left-radius: 10%;
+  margin: 0 auto;
+  padding: 1em;
+  border: 1px solid #ccc;
+}
+
+.form ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.form ul li {
+  margin-top: 1em;
+}
+
+.contact-sec h2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 700;
+  font-stretch: wider;
+  line-height: 52px;
+  position: absolute;
+  width: 65%;
+  height: 56px;
+  color: #fff;
+  top: 123px;
+  left: 66px;
+}
+
+.contact-sec p {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 28px;
+  font-stretch: wider;
+  position: absolute;
+  height: 10rem;
+  width: 90%;
+  top: 188px;
+  left: -12px;
+  color: #ebebff;
+}
+
+.name,
+.email,
+.msg {
+  position: absolute;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  background: #fff;
+}
+
+#name {
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  position: absolute;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  padding: 10px;
+  right: 38px;
+  color: #172b4d;
+  height: 48px;
+  width: 90%;
+  left: 15px;
+  top: 362px;
+}
+
+#email {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  padding: 10px;
+  height: 48px;
+  width: 90%;
+  left: 15px;
+  top: 422px;
+  color: #172b4d;
+}
```

Comment:
Nice job so far. However, you need to match the actual design in terms of the dimensions and spacings. Kindly make sure that your project has all the margins, paddings, etc from the figma design. 

- [ ] There should be spacing between the form inputs which is stuck on each other as shown below.

- [ ]  The position of the submit button should also match the actual design. 

- [ ] Kindly also check and ensure that the distance between the contact form and the `about myself` section is similar to the figma design. 


### Your design 

![Screenshot from 2023-06-09 12-52-11](https://github.com/BilalLiaquat7/Portfolio/assets/65233860/6686c4be-f693-4416-8e4e-55aae9333d11)


### Actual design 

![Screenshot from 2023-06-09 12-51-34](https://github.com/BilalLiaquat7/Portfolio/assets/65233860/6866bb68-e310-4a99-ae36-909e34178126)